lab-tjay/model/minion.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const mongoose = require('mongoose');
+
+const minionSchema = mongoose.Schema({
+  jobTitle: {type:String, required: true},
+  minionRank: {type:String, required: true},
```

Comment:
To be a bit pedantic these should probably just be called `rank` and `age`. Since they are already on the `minion` schema prefixing them with `"minion"` is a bit repetitive.